source/integrations/aws-cli-with-minio.md

```diff
@@ -0,0 +1,83 @@
+# AWS CLI with MinIO Server [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io)
+
+AWS CLI is a unified tool to manage AWS services. It is frequently the tool used to transfer data in and out of AWS S3. It works with any S3 compatible cloud storage service.
+
+In this recipe we will learn how to configure and use AWS CLI to manage data with MinIO Server.
+
+## 1. Prerequisites
+
+Install MinIO Server from [here](https://docs.min.io).
```

Comment:
Need to fix this URL.

https://min.io/docs/minio/linux/reference/minio-server/minio-server.html, I suppose.